.circleci/config.yml

```diff
@@ -195,9 +191,9 @@ jobs:
                 name: Cannon Go 32-bit tests
                 command: |
                   export SKIP_SLOW_TESTS=<<parameters.skip_slow_tests>>
-                  mkdir -p /testlogs
-                  gotestsum --format=testname --junitfile=/tmp/test-results/cannon.xml --jsonfile=/testlogs/log.json \
-                  -- -parallel=8 -coverpkg=github.com/ethereum-optimism/optimism/cannon/... -coverprofile=coverage.out ./...
+                  mkdir -p /tmp/testlogs-<<pipeline.number>>
```

Comment:
Does something need to delete this directory or are we still in a throw-away builder env?

Reply:
As of now this data will stick around. The runners are containerized, so rather than adding cleanup logic to our build jobs I'll instead add a cron job to the runner VM to cycle the containers and clean up any dangling volumes they create.

Reply:
My concern here is that we don't want one job picking up and archiving the logs from another which would be very confusing (and potentially lead to very big storage bills). We could include the build number in the path to make sure it's always unique or would need to delete the dir at the start of the run.

Reply:
Good point. They're prefixed with the build number already, but it's possible that a conflict could occur if a new job with the same build number is scheduled on the same executor. I've updated this to use temp directories inside the build workspace, which is automatically cleaned up between runs.